ui/decredmaterial/switch.go

```diff
@@ -28,8 +38,12 @@ type SwitchButtonText struct {
 	selected                           int
 }
 
-func (t *Theme) Switch(swtch *widget.Bool) Switch {
-	return Switch{material.Switch(t.Base, swtch)}
+func (t *Theme) Switch(swtch *widget.Bool) *Switch {
```

Comment:
Make this a primitive bool variable, handle the `*widget.Bool` here and provide a `Changed()` function.
```suggestion
func (t *Theme) Switch(isChecked bool) *Switch {
```

Reply:
The Switch widget should really just be a skin that accepts an instance of the bool type. This means it shouldn't return an object. Similar to the Button widget that accepts an instance of the Clickable type, the Switch widget should only need to accept an instance of the Bool type.  

To use the switch widget in this current state, we'll need to keep an instance of both the Switch type and the bool type which is quite redundant.